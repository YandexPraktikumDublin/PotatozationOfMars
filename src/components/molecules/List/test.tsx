import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { List } from '.'

describe('<List />', () => {
  const children = 'Test children'

  it('should renders correct <List />', () => {
    const wrapper = shallow(<List>{children}</List>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should has className from className prop', () => {
    const className = 'test-class-name'

    const wrapper = shallow(<List className={className}>{children}</List>)

    expect(wrapper.prop('className')).toMatch(className)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
