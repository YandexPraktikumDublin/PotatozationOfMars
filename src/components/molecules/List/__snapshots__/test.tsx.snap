// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should has className from className prop 1`] = `
<ul
  className="test-class-name"
>
  Test children
</ul>
`;

exports[`<List /> should renders correct <List /> 1`] = `
<ul>
  Test children
</ul>
`;
